src/plugins/vis_type_xy/public/utils/accessors.tsx

```diff
@@ -27,7 +27,7 @@ const getFieldName = (fieldName: string, index?: number) => {
 };
 
 export const isRangeAggType = (type: string | null) =>
-  type === BUCKET_TYPES.DATE_RANGE || type === BUCKET_TYPES.RANGE;
+  type === BUCKET_TYPES.DATE_RANGE || type === BUCKET_TYPES.RANGE || type === BUCKET_TYPES.IP_RANGE;
```

Comment:
please make me a favor. Refactor it to: 
`=> [BUCKET_TYPES.DATE_RANGE, BUCKET_TYPES.RANGE, BUCKET_TYPES.IP_RANGE].includes(type)`